tests/tensorflow/test_sanity_sample.py

```diff
@@ -389,6 +389,7 @@ def get_prepare_checkpoint_configs():
     return config_params
 
 
+@pytest.mark.skip('59320 ticket to resolve not correct testing of prepare_checkpoint')
```

Comment:
@ljaljushkin this test is sanity test and involve only functional testing as far as I know (checkpoints are generating and after eval test is launching with generated checkpoint). You can check test_model_eval https://github.com/openvinotoolkit/nncf/blob/develop/tests/tensorflow/test_sanity_sample.py#L209-L219 which doesn't resume any checkpoint. For accuracy check we employ test_sota_checkpoints

Reply:
discussed, test should be corrected after this PR